src/app/beer_garden/db/mongo/util.py

```diff
@@ -388,3 +389,17 @@ def _should_create_admin():
     # anonymous users, we assume they do not want to re-create the admin
     # user.
     return False
+
+
+def is_in_command_black_list(event):
```

Comment:
> 
> ```python
> def event_blacklisted(event: Event) -> bool:
>     """Docstring goes here"""
>     return _event_blacklisted_by_name(event) or _event_blacklisted_by_command(event)
> ```
> 

I prefer `event_is_blacklisted` over `event_blacklisted` because it makes it clear that you're calling a predicate function. Otherwise, no disagreement.

Reply:
`event_is_blacklisted` is fine with me.  Let's go with that.